### ISSUE TITLE
Suggestion: option to lock/schedule customize app button

### ISSUE THREAD

**Issue by abhay-s** [sduduzog/slim-launcher#137]
Sir,
Can you please add option to lock customize apps button so that one could get it locked by someone and get rid of all the distracting apps(like parental control) . (It could also act as a privacy tool for someone who wants to hand over phones to others with certain app ...maybe)
OR
(Unnecessary here maybe)You could make a scheduler and lock time somehow so that one could open those app for a particular time period scheduled previously(probably something like in your new app). 

in short,
Whatever be the case please allow it to lock the apps from being changed forever or for any certain time . 
Hope you consider the suggestion 😃

**Comment by sduduzog** [comment]
I see where this is useful, but I fear edge cases would not be fun to deal with, also this could open up room for abuse also. Another thing is, I try to use little to no permissions at all as much as possible just so that I maintain trust with consumers... as I don't really need more.

I feel like the app direction is so close to the sweet spot, the very issue we're trying to tackle...
